src/utils/helpers/list/findUserRole.ts

```diff
@@ -16,9 +16,7 @@ type UserRoles = {
 
 const userRoles: UserRoles = {
   applicant: [PermissionPolicyType.Apply],
-  consolidator: [PermissionPolicyType.Assign, PermissionPolicyType.Review],
-  reviewer: [PermissionPolicyType.Review],
-  assigner: [PermissionPolicyType.Assign],
+  reviewer: [PermissionPolicyType.Review, PermissionPolicyType.Assign],
```

Comment:
I think this will be checking if both match! It might be needed to update the code that verifies this to use an `or` statement